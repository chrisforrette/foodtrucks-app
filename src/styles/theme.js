// Breakpoints

export const breakpoints = {
  small: '600px',
  medium: '900px',
  large: '1200px',
  xlarge: '1800px'
}

// Fonts

// const fontHelvetica = '"HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif'
const fontHelveticaLight = '"HelveticaNeue-Light", "Helvetica Neue Light", "Helvetica Neue", Helvetica, Arial, sans-serif'
const fontHelveticaBold = '"HelveticaNeue-Bold", "Helvetica Neue Bold", "Helvetica Neue", Helvetica, Arial, sans-serif'

export const fonts = {
  heading: {
    primary: fontHelveticaBold
  },
  body: {
    primary: fontHelveticaLight
  }
}

// Colors

export const colors = {
  // Grey
  darkGrey: '#333',
  grey: '#999',
  lightGrey: '#eee'
}

export default {
  breakpoints,
  fonts,
  colors
}
