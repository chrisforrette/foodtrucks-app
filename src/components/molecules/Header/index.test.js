import React from 'react'
import { shallow } from 'enzyme'
import toJson from 'enzyme-to-json'
import { Header } from './index'
import theme from '../../../styles/theme'

it('should render', () => {
  const rendered = shallow(<Header
    theme={theme}
    dispatch={() => {}}
    loading={false}
    mapBoundsChanged={false} />)
  expect(toJson(rendered)).toMatchSnapshot()
})

it('should render a loading state', () => {
  const rendered = shallow(<Header
    theme={theme}
    dispatch={() => {}}
    loading={true}
    mapBoundsChanged={false} />)
  expect(toJson(rendered)).toMatchSnapshot()
})

it('should render visibly when map bounds are changed', () => {
  const rendered = shallow(<Header
    theme={theme}
    dispatch={() => {}}
    loading={false}
    mapBoundsChanged={true} />)
  expect(toJson(rendered)).toMatchSnapshot()
})
