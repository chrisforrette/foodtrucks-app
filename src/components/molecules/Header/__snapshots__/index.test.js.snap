// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render 1`] = `
<styled.header>
  <styled.div>
    <Styled(LoadingIndicator)
      fade={true}
      isLoading={false}
    />
    <styled.h1>
      Food Truck Finder
    </styled.h1>
  </styled.div>
  <styled.div>
    <Component
      disabled={false}
      hide={true}
      onClick={[Function]}
    >
      Search This Area
    </Component>
  </styled.div>
</styled.header>
`;

exports[`should render a loading state 1`] = `
<styled.header>
  <styled.div>
    <Styled(LoadingIndicator)
      fade={true}
      isLoading={true}
    />
    <styled.h1>
      Food Truck Finder
    </styled.h1>
  </styled.div>
  <styled.div>
    <Component
      disabled={true}
      hide={true}
      onClick={[Function]}
    >
      Search This Area
    </Component>
  </styled.div>
</styled.header>
`;

exports[`should render visibly when map bounds are changed 1`] = `
<styled.header>
  <styled.div>
    <Styled(LoadingIndicator)
      fade={true}
      isLoading={false}
    />
    <styled.h1>
      Food Truck Finder
    </styled.h1>
  </styled.div>
  <styled.div>
    <Component
      disabled={false}
      hide={false}
      onClick={[Function]}
    >
      Search This Area
    </Component>
  </styled.div>
</styled.header>
`;
